src/datashare/azext_datashare/manual/_help.py

```diff
@@ -0,0 +1,618 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['datashare consumer'] = """
+    type: group
+    short-summary: Commands for consumers to manage datashare
+"""
+
+helps['datashare account'] = """
+    type: group
+    short-summary: Commands to manage datashare accounts
+"""
+
+helps['datashare account list'] = """
+    type: command
+    short-summary: List datashare accounts
+    examples:
+      - name: List accounts by resource group
+        text: |-
+               az datashare account list --resource-group MyResourceGroup
+"""
+
+helps['datashare account show'] = """
+    type: command
+    short-summary: Show an account
+    examples:
+      - name: Show account information
+        text: |-
+               az datashare account show --name MyAccount --resource-group MyResourceGroup
+"""
+
+helps['datashare account create'] = """
+    type: command
+    short-summary: Create an account
+    parameters:
+      - name: --identity
+        short-summary: Identity of resource.
+        long-summary: |
+            Usage:   --identity [type=SystemAssigned] [tenantId=VAL principalId=VAL]
+    examples:
+      - name: Create an account
+        text: |-
+               az datashare account create --identity type=SystemAssigned --location "West US 2" --tags tag1=Red tag2=White --name MyAccount --resource-group MyResourceGroup
+"""
+
+helps['datashare account update'] = """
+    type: command
+    short-summary: Patch an account
+    examples:
+      - name: Update account tags
+        text: |-
+               az datashare account update --name MyAccount --tags tag1=Red tag2=White --resource-group MyResourceGroup
+"""
+
+helps['datashare account delete'] = """
+    type: command
+    short-summary: Delete an account
+    examples:
+      - name: Delete the account
+        text: |-
+               az datashare account delete --name MyAccount --resource-group MyResourceGroup
```

Comment:
just curious, is resource group required when deleting? can usre passs a datashare account resource id for deleting?

Reply:
Yes. users can use `--ids` to pass one or multiple resource ids, then no need for `--name` and `--resource-group`.